cylc/flow/prerequisite.py

```diff
@@ -223,6 +224,45 @@ def dump(self):
         # (Else trigger wiped out by pre-initial simplification.)
         return res
 
+    def api_dump(self, workflow_id):
+        """Return list of populated Protobuf data objects."""
+        if self.conditional_expression:
+            temp = self.get_raw_conditional_expression()
+            temp = temp.replace('|', ' | ')
+            temp = temp.replace('&', ' & ')
+        else:
+            for s_msg in self.satisfied:
+                temp = self.MESSAGE_TEMPLATE % s_msg
+        conds = []
+        num_length = int(math.ceil(float(len(self.satisfied)) / float(10)))
```

Comment:
Same as `num_length = math.ceil(len(self.satisfied) / 10)`?

I think [math.ceil](https://docs.python.org/3/library/math.html#math.ceil) will return an integer.

And in Python 3 the division of two integers will always return a float now (which was different in Python 2).

Reply:
Fixed (inherited from method above it on clone).